### DISCUSSION TITLE
唯一模型的统计性能怎么优化

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#14183] [A - General / Q&A] [answered]
唯一模型不能预聚合，那么实时统计，效果怎么样？
目前统计4500W条 需要1秒左右。
<img width="571" alt="image" src="https://user-images.githubusercontent.com/32891043/201249724-d8ca1fa0-50a5-4e5b-a99e-4df04330123d.png">

**Comment by zy-kkk** [comment, marked as answer]
你好，在我们即将推出的1.2版本中，增加了unique模型的merge on write实现，在聚合查询场景有很大优势，文档介绍https://doris.apache.org/zh-CN/docs/dev/data-table/data-model
针对你目前的话，可以尝试调整parallel_fragment_exec_instance_num参数试一下
set global parallel_fragment_exec_instance_num=2。具体数值最大不要超过CPU核数的一半，过大影响并发

**Reply by a3225167** [reply]
好的，谢谢，merge on write实现会不会影响写入哦？

**Reply by zy-kkk** [reply]
不会，优化的就是这部分，大量写入情况下性能比merge on read好很多
